src/packages/@app/pages/Collections/CollectionsPage.svelte

```diff
@@ -279,6 +301,18 @@
                       <FolderExplorerPage tab={$activeTab} />
                     </div>
                   </Motion>
+                  {:else if $activeTab?.type === ItemType.WORKSPACE}
+                  <Motion {...scaleMotionProps} let:motion>
+                    <div use:motion>
+                      <WorkspaceExplorerPage
+                      updateDescription={updateCurrentWorkspaceDesc}
+                        {collectionList}
+                        handleChange={updateCurrentWorkspace}
```

Comment:
Can we some meaningful names here like onUJpdateWorkspaceName, same for description also